acceleration/distributed_training/brats_training_ddp.py

```diff
@@ -22,31 +22,28 @@
 
 Main steps to set up the distributed training:
 
-- Execute `torch.distributed.launch` to create processes on every node for every GPU.
+- Execute `torchrun` to create processes on every node for every GPU.
   It receives parameters as below:
   `--nproc_per_node=NUM_GPUS_PER_NODE`
   `--nnodes=NUM_NODES`
-  `--node_rank=INDEX_CURRENT_NODE`
   `--master_addr="localhost"`
   `--master_port=1234`
-  For more details, refer to https://github.com/pytorch/pytorch/blob/master/torch/distributed/launch.py.
+  For more details, refer to https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py.
```

Comment:
The correct link is as follows.
https://github.com/pytorch/pytorch/blob/main/torch/distributed/run.py